src/forms/EditUser.js

```diff
@@ -1,36 +1,49 @@
-import React, { useState, useEffect } from 'react'
```

Comment:
This file seems that is not necessary, can you check, and if not, you can remove it and all its imports